frontend/src/storybook/encodedAssets.ts

```diff
@@ -0,0 +1,10 @@
+/*
```

Comment:
Use hyphen-case instead of camelCase in the frontend. I've split the file into two (diagram.ts and model-badge.ts)